src/components/RealtimeLockdrop/ClaimStatus.tsx

```diff
@@ -169,11 +171,10 @@ const ClaimStatus: React.FC<Props> = ({
         // unsubscribe flag
         let isUnmounting = false;
         const balanceSub = async () => {
-            const unsub = await plasmApi.query.system.account(plasmAddr, ({ data: balance }) => {
+            const unsub = await api.query.system.account(plasmAddr, ({ data: balance }) => {
                 if (isUnmounting) unsub();
                 const freeBal = balance.free;
-                const plmTokens = plasmUtils.femtoToPlm(new BigNumber(freeBal.toString(10))).toFixed(3);
-                const formatBal = parseFloat(plmTokens).toLocaleString('en');
+                const formatBal = formatBalance(freeBal, { decimals: tokenDecimals, withSi: true });
```

Comment:
The formatted string balance says 'Has balance of <balance> <formatted unit> PLM' like below.

![Screen Shot 2021-08-18 at 2 44 07 PM](https://user-images.githubusercontent.com/40356749/129844079-c5ddc0d2-4640-4474-a13f-7bb85b421956.png)

But it should say, for example, `190.1843 MPLM` instead of `190.1843 MUnit PLM`
![Screen Shot 2021-08-18 at 2 49 34 PM](https://user-images.githubusercontent.com/40356749/129844418-4d29c467-0abf-4c0f-8a09-47e1631e1f17.png)

(perhaps using `formatBalance.setDefaults({ unit: 'PLM' });` may fix it?)

You may want to remove this line too
https://github.com/PlasmNetwork/lockdrop-ui/blob/3e311ee81935f2b95db50beb8c146fcfb6e2514d/src/components/RealtimeLockdrop/ClaimStatus.tsx#L359
and